### DISCUSSION TITLE
Als Elternteil möchte ich für alle wichtigen Termine eine Benachrichtigung erhalten, um immer auf den neusten Stand meines Kindes zu sein.

### DISCUSSION THREAD

**Discussion by CodeRider101** [SE-TINF22B2/G2-Schoolify#24] [User Stories]
### Beschreibung:
Wenn wichtige Termine (z.B. Elternsprechtag) eingetragen werden (für die es meist einen Elternbrief gibt) sollen Eltern eine Mail bekommen, um darüber benachrichtigt zu werden. 

**Optionale Erweiterungen:**
- Zusätzlich kann ein Datum festgelegt werden, an welchem die Mails versendet werden
- Erinnerungsmails z.B. eine Woche vorher
- Elternbriefe direkt in die Mail einfügen

### Akzeptanz-Kriterien:
- [ ] Benachrichtigungen die Eltern per Mail sobald wichtige Termine eingetragen werden

**Comment by laralngn** [comment]
Gibt es dann auch eigene Accounts für die Eltern oder teilen die sich die Schüleraccounts?

**Reply by CodeRider101** [reply]
Keinen eigenen Account sondern man speichert die Mail-Adressen der Eltern beim Kind ab und die Eltern bekommen e-Mail notifikationen.

**Comment by two-Gee** [comment]
nicht mehr in requirements
